.github/workflows/main.yaml

```diff
@@ -231,6 +234,7 @@ jobs:
       # However, dind container ends up having / folder mounted on overlay
       # filesystem, whereas /__w which contains Cobalt source code is on tmpfs.
       TMPDIR: /__w/_temp
+    timeout-minutes: 90
```

Comment:
This can be set to 60 minutes here. The 99.9 and 99.99 percentile durations for the past 3 months shown here:

![image](https://github.com/youtube/cobalt/assets/503316/c5fdd28b-daa1-44d1-aa5c-a3cdbfcb1471)

And these times are mostly without the cached gradle install you just added.

Reply:
Ack, reduced. Seems like we have enough buffer